gapic/generator/generator.py

```diff
@@ -217,9 +217,10 @@ def _generate_samples_and_manifest(
 
         if index.metadata_index.snippets:
             # NOTE(busunkim): Not all fields are yet populated in the snippet metadata.
-            # Expected filename: snippet_metadata_{apishortname}_{apiversion}.json
+            # Expected filename: snippet_metadata.{proto_package}.json
+            # For example: snippet_metadata_google.cloud.aiplatform.v1.json
             snippet_metadata_path = str(pathlib.Path(
-                out_dir) / f"snippet_metadata_{api_schema.naming.name}_{api_schema.naming.version}.json").lower()
+                out_dir) / f"snippet_metadata_{api_schema.naming.proto_package}.json").lower()
```

Comment:
```suggestion
                out_dir) / f"snippet_metadata.{api_schema.naming.proto_package}.json").lower()
```
Minor suggestion to match the format of nodejs here https://github.com/googleapis/nodejs-language/blob/main/samples/generated/v1/snippet_metadata.google.cloud.language.v1.json

Reply:
As per an offline discussion, the changes proposed by @dizcology are correct.